src/ceph-volume/ceph_volume/devices/lvm/prepare.py

```diff
@@ -297,10 +297,29 @@ def prepare(self):
             'ceph.osdspec_affinity': prepare_utils.get_osdspec_affinity()
         }
         if self.args.filestore:
-            #TODO: allow auto creation of journal on passed device, only works
-            # when physical device is passed, not LV
             if not self.args.journal:
-                raise RuntimeError('--journal is required when using --filestore')
+                logger.info(('no journal was specifed, creating journal lv '
+                            'on {}').format(self.args.data))
```

Comment:
Unfortunately we need to avoid format strings for now. At least some jenkins machines are running py3.5 and will fail with invalid syntax on format strings. Also this makes me unsure about py versions in some supported OS'.

Reply:
Oh, ok.